both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
+    return wca.roundByCode[this.roundCode];
+  },
+  eventName: function() {
+    return this.eventCode ? wca.eventByCode[this.eventCode].name : '';
```

Comment:
For non competition rounds. Which BTW I think should be their own table/class, but that's not for now.

I don't understand asserts in production code, but I can add whatever you think should be there.